code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -376,3 +376,10 @@
 	desc = ""
 	icon_state = "doom"
 	item_state = "doom"
+
+/obj/item/clothing/shoes/workboots
+	name = "Workboots"
+	desc = "Thick-soled boots for industrial work environments."
+	icon_state = "workboots"
+	item_state = "workboots"
+	species_fit = list(VOX_SHAPED)
```

Comment:
They probably shouldn't fit Vox, since Vox already can't wear normal shoes, and it makes no sense that they could wear heavy duty boots.